### DISCUSSION TITLE
When using cmake -j for LLVM, monitor shut down

### DISCUSSION THREAD

**Discussion by Haoban** [halide/Halide#4709] [General]
There is no response for my monitor, when I use cmake -j command for build LLVM project!
Does anyone met this before?

**Comment by abadams** [comment]
If your computer straight-up turns off, this can be caused by lots of things. E.g. your CPU not getting enough current when all cores turbo up. That's unlikely for an llvm build though, because it's not turning on much of the CPU (e.g. the vector units). If things are just freezing up on you, then it's more likely a flavor of thrashing - running out of physical ram.

Either way, I'd use -j8 instead of -j, or some other small number.
